scripts/format.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+  # format source files
+  yapf -i -r securedrop bin
```

Comment:
I just tested the script for the first time, and I think it would be good to use `-q` (quiet) flag here. We don't really care about the formatting errors found when we are formatting all files. However, when using `check` option, we do, so we don't need `-q` there.

Reply:
I am not seeing the -q flag in the documentation. Can you point me in the right direction.

Reply:
```bash
$ yapf --help
```

```
usage: yapf [-h] [-v] [-d | -i | -q] [-r | -l START-END] [-e PATTERN] [--style STYLE] [--style-help] [--no-local-style] [-p] [-vv] [files [files ...]]

Formatter for Python code.

positional arguments:
  files                 reads from stdin when no files are specified.

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show version number and exit
  -d, --diff            print the diff for the fixed source
  -i, --in-place        make changes to files in place
  -q, --quiet           output nothing and set return value
  -r, --recursive       run recursively over directories
  -l START-END, --lines START-END
                        range of lines to reformat, one-based
  -e PATTERN, --exclude PATTERN
                        patterns for files to exclude from formatting
  --style STYLE         specify formatting style: either a style name (for example "pep8" or "google"), or the name of a file with style settings. The default is pep8 unless a .style.yapf or setup.cfg file
                        located in the same directory as the source or one of its parent directories (for stdin, the current directory is used).
  --style-help          show style settings and exit; this output can be saved to .style.yapf to make your settings permanent
  --no-local-style      don't search for local style definition
  -p, --parallel        run yapf in parallel when formatting multiple files. Requires concurrent.futures in Python 2.X
  -vv, --verbose        print out file names while processing
```

Reply:
Weird. I am suprised I missed that last night.

Reply:
I am also getting this output from the terminal:
"usage: yapf [-h] [-v] [-d | -i | -q] [-r | -l START-END] [-e PATTERN] [--style STYLE] [--style-help] [--no-local-style] [-p] [-vv] [files [files ...]]
yapf: error: argument -q/--quiet: not allowed with argument -i/--in-place"

Reply:
Oh... that's mildly annoying. In that case, redirect `stderr` to `/dev/null`.